ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezTask.java

```diff
@@ -253,7 +259,7 @@ public int execute() {
           counters = mergedCounters;
         } catch (Exception err) {
           // Don't fail execution due to counters - just don't print summary info
-          LOG.warn("Failed to get counters. Ignoring, summary info will be incomplete. " + err, err);
+          LOG.warn("Failed to get counters. Ignoring, summary info will be incomplete.", err);
```

Comment:
Missing {} here?

Reply:
Nope.  Removing this was intentional.  SLF4J will log the entire Exception information, no need to print it twice.